### PR TITLE
U.S. Bank Treasury Management (Verified Publisher)

### PR DESCRIPTION
**Pull request by jzworkman**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by jzworkman** [comment]
@microsoft-github-policy-service agree [company="U.S. Bank"]

**Comment by jzworkman** [comment]
@microsoft-github-policy-service agree company="U.S. Bank"

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2778 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "U.S. Bank Treasury Management",
+    "description": "U.S. Bank payments and treasury banking connector for Microsoft power platform users.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:6]: @jzworkman Is it possible to remove "Microsoft power platform" from the description. it should meet our requirements. please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2778 @@
...
+              "description": "ISO 8601 format",
+              "example": "2020-05-03T00:00:00.000Z"
+            }
+          }
+        }
+      ]
+    },
+    "Checks": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:2409]: @jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2778 @@
...
+      }
+    },
+    "lastRefreshedDateTime": {
+      "type": "string",
+      "description": "ISO 8601 Date format that the data is from.",
+      "example": "2020-05-03T00:00:00.000Z"
+    },
+    "ACH": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:2425]: @jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2778 @@
...
+          "type": "string",
+          "format": "date",
+          "description": "Effective entry date of the transaction",
+          "example": 200130
+        }
+      }
+    },
+    "RTP": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:2467]: @jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2778 @@
...
+        "creditorAddress": {
+          "type": "string",
+          "description": "Creditor Address",
+          "example": "9955 SE Washington St Portland 97216"
+        }
+      }
+    },
+    "Zelle": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:2539]: @jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2778 @@
...
+        "parameterValue": {
+          "type": "string",
+          "description": "Parameter Value",
+          "example": "abc123"
+        }
+      }
+    },
+    "Wires": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:2581]: @jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".

File: certified-connectors/U.S. Bank Treasury Management/readme.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+## U.S. Bank
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/readme.md:1]: @jzworkman this great readme file. is it possible to include all required sections from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). you can see the code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@jzworkman thank you so much for working hard on this. I had some small feedback on the PR. but everything is looking great, all operations are good to go.
Thank you so much again.

**Comment by jzworkman** [comment]
> @jzworkman thank you so much for working hard on this. I had some small feedback on the PR. but everything is looking great, all operations are good to go. Thank you so much again.

Thanks @Amjed-Ayoub Changes have all been made(and caught a few others during my review).  Apologies for the other readme format, the team was told that was just an example file and not a require template.  Have updated the formatting to follow that template now

File: certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,2778 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "U.S. Bank Treasury Management",
+    "description": "U.S. Bank payments and treasury banking APIs.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json:6]: @jzworkman thank you so much for fixing the connector files. Unfortunately, the word "API" is not allowed in the description too. Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector). 
Thank you so much again.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @jzworkman,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**